examples/pgconfig/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+appVersion: '1.0'
+description: Example for geoserver-cloud with pgconfig
+name: gs-cloud-pgconfig
+version: 0.1.0
+dependencies:
+  - name: geoservercloud
+    repository: file://../../
+    version: 0.1.0
+  # Postgres dependency used for pgconfig/catalog database
+  - name: 'postgresql'
+    version: 14.0.0
```

Comment:
`pgconfig` requires postgres 15+

Reply:
It's the bitnami helmchart release, not the Postgres one. Release 14 includes Postgres 16.x.
https://github.com/bitnami/charts/tree/main/bitnami/postgresql#upgrading